src/System.Security.Permissions/ref/System.Security.Permissions.csproj

```diff
@@ -1,11 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{07CAF142-B259-418E-86EF-C4BD8B50253E}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <Configurations>net461-Debug;net461-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release</Configurations>
+    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true' or '$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'">true</IsPartialFacadeAssembly>
+    <Configurations>net461-Debug;net461-Release;netfx-Debug;netfx-Release;netstandard-Debug;netstandard-Release;netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Security.Permissions.cs" />
+    <Compile Include="System.Security.Permissions.netstandard.cs" Condition="'$(TargetsNetFx)' != 'true' and '$(TargetGroup)' == 'netstandard'" />
+    <Compile Include="System.Security.Permissions.netcoreapp.cs" Condition="'$(TargetGroup)' == 'netcoreapp'" />
+    <Compile Include="System.Security.Permissions.netcoreapp.cs" Condition="'$(TargetGroup)' == 'uap'" />
```

Comment:
I'll clean up this part after I find the correct condition to fix the build issues